tests/migration_test.go

```diff
@@ -2112,41 +2112,54 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				cfg := getCurrentKv()
 				var timeout int64 = 5
 				cfg.MigrationConfiguration = &v1.MigrationConfiguration{
-					ProgressTimeout:         &timeout,
 					CompletionTimeoutPerGiB: &timeout,
 					BandwidthPerMigration:   resource.NewMilliQuantity(1, resource.BinarySI),
 				}
 				tests.UpdateKubeVirtConfigValueAndWait(cfg)
 			})
-			PIt("[test_id:2227] should abort a vmi migration without progress", func() {
-				vmi := tests.NewRandomFedoraVMIWithGuestAgent()
-				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1Gi")
+			Context("without progress", func() {
 
-				By("Starting the VirtualMachineInstance")
-				vmi = runVMIAndExpectLaunch(vmi, 240)
+				BeforeEach(func() {
+					cfg := getCurrentKv()
+					var timeout int64 = 5
+					cfg.MigrationConfiguration = &v1.MigrationConfiguration{
+						ProgressTimeout:         &timeout,
+						CompletionTimeoutPerGiB: &timeout,
+						BandwidthPerMigration:   resource.NewMilliQuantity(1, resource.BinarySI),
+					}
+					tests.UpdateKubeVirtConfigValueAndWait(cfg)
+				})
 
-				By("Checking that the VirtualMachineInstance console has expected output")
-				Expect(console.LoginToFedora(vmi)).To(Succeed())
+				It("[test_id:2227] should abort a vmi migration without progress", func() {
```

Comment:
nit: remove `without progress` from description since it is already specified in the Context